scripts/hotkey-notes.lua

```diff
@@ -0,0 +1,12 @@
+-- prints info on assigned hotkeys to the console
+
+local h_list = {'F1', 'F2', 'F3', 'F4', 'F5', 'F6', 'F7', 'F8',
+                'Shift+F1', 'Shift+F2', 'Shift+F3', 'Shift+F4',
+				'Shift+F5', 'Shift+F6', 'Shift+F7', 'Shift+F8'}
+
```

Comment:
It would be more correct to use something like
```lua
dfhack.screen.getKeyDisplay(df.interface_key.D_HOTKEY1)
```
here, to account for possible changes to keybindings. In fact, since the binding IDs are in order, you could get rid of `h_list` entirely and replace `h_list[i]` with this:
```lua
dfhack.screen.getKeyDisplay(df.interface_key.D_HOTKEY1 + i - 1)
```